t5/data/mixtures.py

```diff
@@ -25,24 +25,49 @@
 from t5.data.utils import rate_unsupervised
 import tensorflow_datasets as tfds
 
-
+# These weights are based on the number of examples in each dataset.
+_SUPER_GLUE_WEIGHT_MAPPING = {
+    "dpr_v001_simple": 1_322.,
+    "super_glue_wsc_v102_simple_train": 554.,
+    "super_glue_wsc_v102_simple_eval": 104.,
```

Comment:
WSC simple eval has no train split so it's probably clearer to set this to 0. https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/data/tasks.py#L273